packages/content-management/table_list_view_table/src/components/user_filter_panel.tsx

```diff
@@ -0,0 +1,177 @@
+    // attach null to the end of the list to represent the "no creator" option
+    const users = [...query.data, null];
+
+    if (!searchTerm) {
+      return users;
+    }
+
+    return users.filter((user) => {
+      // keep the "no creator" option if it's selected
+      if (!user) {
+        return selectedUsers.includes(NULL_USER);
+      }
+
+      // keep the user if it's selected
+      if (selectedUsers.includes(user.uid)) return true;
+
+      // filter only users that match the search term
+      const searchString = (
+        user.uid +
+        user.user.username +
+        (user.user.email ?? '') +
+        (user.user.full_name ?? '')
+      ).toLowerCase();
+      return searchString.includes(searchTerm.toLowerCase());
+    });
+  }, [query.data, searchTerm, selectedUsers]);
+
+  const noUsersTip = (
+    <EuiIconTip
+      aria-label="Additional information"
+      position="bottom"
+      type="questionInCircle"
+      color="inherit"
+      iconProps={{ style: { verticalAlign: 'text-bottom', marginLeft: 2 } }}
+      css={{ textWrap: 'balance' }}
+      content={
+        <FormattedMessage
+          id="contentManagement.tableList.listing.userFilter.emptyMessageTooltip"
+          defaultMessage="Creators are assigned when dashboards are created, (for all dashboards created after version 8.14)."
+        />
+      }
+    />
+  );
+
+  return (
+    <>
+      <UserProfilesPopover
+        button={
+          <EuiFilterButton
+            iconType="arrowDown"
+            iconSide="right"
+            data-test-subj="userFilterPopoverButton"
+            onClick={() => setPopoverOpen(!isPopoverOpen)}
+            hasActiveFilters={selectedUsers.length > 0}
+            numActiveFilters={selectedUsers.length}
+            grow
+          >
+            <FormattedMessage
+              id="contentManagement.tableList.listing.userFilter.filterLabel"
+              defaultMessage="Created by"
+            />
+          </EuiFilterButton>
+        }
+        isOpen={isPopoverOpen}
+        closePopover={() => setPopoverOpen(false)}
+        selectableProps={{
+          'data-test-subj': 'userSelectableList',
+          isLoading: query.isLoading,
+          options: visibleOptions,
+          errorMessage: query.error ? (
+            <FormattedMessage
+              id="contentManagement.tableList.listing.userFilter.errorMessage"
+              defaultMessage="Failed to load users"
+            />
+          ) : undefined,
+          emptyMessage: (
+            <p>
+              <FormattedMessage
+                id="contentManagement.tableList.listing.userFilter.emptyMessage"
+                defaultMessage="None of the dashboards have creators"
+              />
+              {noUsersTip}
+            </p>
+          ),
+          nullOptionLabel: i18n.translate(
+            'contentManagement.tableList.listing.userFilter.noCreators',
+            {
+              defaultMessage: 'No creators',
+            }
+          ),
```

Comment:
Should this be filtered out when there are no Dashboards without a creator? 

![Apr-26-2024 11-19-52](https://github.com/elastic/kibana/assets/8698078/e348ec6f-2b86-4fe2-bd98-7042c27cd4ad)

Reply:
good point, I think it should, will work on that